app/views/posts/show.html.erb

```diff
@@ -1 +1,16 @@
-<div>Here is a the post you are looking for</div>
\ No newline at end of file
+<div class="post-ctn">
+  <div class="post-info-ctn">
+    <div class="post-info-head">
+      <h3>Post <%=@post.id %> by <%= @user.name %></h3>
+      <div class="post-right-info">
+        <h5>Comment : x</h5>
+        <h5>Likes : x</h5>
+      </div>
+    </div>
+
+    <p><%= @post.text %></p>
+  </div>
```

Comment:
- [ ] Kindly use our  `recent_comments` method in the post show view to render the most recent comments.

Reply:
Hi @elisha2kyakpo1 , sorry, I am not getting what you are talking about. We never created any `recent_comment` method. Only a method to get the three recent posts and I am using that one in my code.

Reply:
@kosher9, you have a function called `five_recent_comments` in the post model. That function isn't used anywhere.

Reply:
Oh you are right. Let me fix that.

Reply:
@elisha2kyakpo1 , I am done. Should I request another review?